src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java

```diff
@@ -0,0 +1,10 @@
+package com.agilethought.atsceapi.exception;
+
+public class ErrorMessage {
+
+    public static final String badRequestMessage = "The field ";
```

Comment:
All contans name must be named in UpperCase separated by _. Example: `BAD_REQUEST_MESSAGE`